.changelog/2817.txt

```diff
@@ -0,0 +1,23 @@
+```release-note:enhancement
+resource/mongodbatlas_cluster: Adds `pinned_fcv` attribute
+```
+
+```release-note:enhancement
+data-source/mongodbatlas_cluster: Adds `pinned_fcv` attribute
+```
+
+```release-note:enhancement
+data-source/mongodbatlas_clusters: Adds `pinned_fcv` attribute
+```
+
+```release-note:bug
+resource/mongodbatlas_cluster: `mongo_db_major_version` attribute is populated with binary version when FCV pin is active
```

Comment:
this is related to upstream fix [CLOUDP-269950](https://jira.mongodb.org/browse/CLOUDP-269950) that was applied to 2024-08-05 API.